medallion/test/test_mongodb_backend.py

```diff
@@ -426,3 +430,193 @@ def test_marking_defintions(self):
         assert manifests["objects"][0]["id"] == "marking-definition--34098fce-860f-48ae-8e50-ebd3cc5e41da"
         self.assertEqual(len(manifests["objects"]), 1, "Expected exactly one result")
         # ------------- END: get objects section 4 ------------- #
+
+    def test_get_collections_401(self):
```

Comment:
When tests are duplicated between backends, it might be better to just define them in a base class, and have the MemoryBackendTests and MongoBackendTests only change what needs to be changed to test the various backends. Otherwise it's possible that the two files will drift. We can fix that after this PR, though.